diff --git a/rally/cmd/cliutils.py b/rally/cmd/cliutils.py
index 896b141..e35898b 100644
--- a/rally/cmd/cliutils.py
+++ b/rally/cmd/cliutils.py
@@ -433,6 +433,8 @@ def run(argv, categories):
                                      handler=parser)
 
     CONF.register_cli_opt(category_opt)
+    CONF.register_cli_opt(cfg.ListOpt("plugin-path",
+                          help="Custom plugins location"))
 
     try:
         CONF(argv[1:], project="rally", version=version.version_string())
@@ -511,6 +513,8 @@ def run(argv, categories):
         utils.load_plugins("/opt/rally/plugins/")
         utils.load_plugins(os.path.expanduser("~/.rally/plugins/"))
         utils.import_modules_from_package("rally.plugins")
+        for path in CONF.plugin_path or []:
+            utils.load_plugins(path)
 
         validate_deprecated_args(argv, fn)
 
